index.html

```diff
@@ -197,7 +197,96 @@ <h2 class="seventh-sheet__contact-me">Contact me</h2>
       </div>
     </form>
       <div class="end-line"><img src="./images/end-line.png" alt="end-line"></div>
-  </section>
+  </section><!--
+  <section class="desktop" id="desktop-modal">
+    <section class="desktop__area">
+      <article>
+        <div class="desktop-heading">
+          <div class="desktop-title">
+            <h2 class="desktop__area-title" id="desktop-project">Tonic</h2>
+            <a class="close" id="closePopup" href="#">&times;</a>
+          </div>
+          <div class="desktop__area-frame">
+            <h4 class="desktop__area-frame-canopy">CANOPY</h4>
+            <img class="desktop__area-frame-dot" src="./images/dot.png" alt="dot">
+            <h4 class="desktop__area-frame-BackEndDev">Back End Dev</h4>
+            <img class="desktop__area-frame-dot" src="./images/dot.png" alt="dot">
+            <h5 class="desktop__area-frame-BackEndDev">2015</h5>
+          </div>
+          <img class="desktop__area-big-image" src="./images/tonic.svg" alt="project_tonic" id="detailImage">
+        </div>
+        <div class="desktop-footer">
+          <p class="desktop__paragraph" id="desktop-modal">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+          <div class="desktop-footer-buttons" id="desktop-technology">
+            <ul id="detailLanguajes">
+              <li>html</li>
+              <li>css</li>
```

Comment:
- [ ] [Optional] Kindly remove the commented old pieces of code as they will make your project look messy. Remember if you need to review a previous version of your code, you can always use git history

Reply:
I have spent 1 hour looking for the reason my code is totally broken and I haven't been able so far. Sirs, my code is off...

Reply:
...it is too much time wasted, this is going to affect my performance...